src/plugins/spotifyControls/index.tsx

```diff
@@ -77,6 +77,13 @@ export default definePlugin({
                 match: /repeat:"off"!==(.{1,3}),/,
                 replace: "actual_repeat:$1,$&"
             }
+        },
+        {
+            find: 'P.SpotifyResourceTypes.TRACK',
+            replacement: {
+                match: /\(0,(.{1,2})\.isNotNullish\)\((.{1,2})\.id\)&&/,
```

Comment:
Use \i for matching identifiers

```suggestion
                match: /\(0,(\i)\.isNotNullish\)\((\i)\.id\)&&/,
```